modules/aggregation-layers/src/contour-layer/contour-layer.ts

```diff
@@ -16,11 +16,11 @@ import {
   DefaultProps
 } from '@deck.gl/core';
 import {PathLayer, SolidPolygonLayer} from '@deck.gl/layers';
-import {WebGLAggregator} from '../aggregation-layer-v9/gpu-aggregator/webgl-aggregator';
-import {CPUAggregator} from '../aggregation-layer-v9/cpu-aggregator/cpu-aggregator';
-import AggregationLayer from '../aggregation-layer-v9/aggregation-layer';
-import {AggregationLayerProps} from '../aggregation-layer-v9/aggregation-layer';
-import {AggregationOperation} from '../aggregation-layer-v9/aggregator';
+import {WebGLAggregator} from '../common/gpu-aggregator/webgl-aggregator';
```

Comment:
Nit: `common` is not an exciting directory name,  Consider splitting into `aggregator` and `aggregation-layer` directories.

One reason why I think this is that the two files are on quite different level of abstractions (i.e. I see the `Aggregator` as a primary export that we would like to continue to make useful for aggregation compute independently of layers.